helper_windows_test.go

```diff
@@ -116,3 +116,8 @@ func copyTestdataFileTo(src, dest string) []string {
 func singleCommandNoArgs(command string) []string {
 	return []string{command}
 }
+
+func shouldRunAdminTests() bool {
```

Comment:
Might be worth adding a comment here to explain why this is opt-out rather than opt-in (to encourage future developers not to change to opt-in with realising the potential consequences).